docs/core/install/linux-package-manager-centos7.md

```diff
@@ -55,3 +55,11 @@ sudo yum install dotnet-runtime-3.1
 ## How to install other versions
 
 [!INCLUDE [package-manager-switcher](./includes/package-manager-heading-hack-pkgname.md)]
+
+## Troubleshoot the package manager
+
+This section provides information on common errors you may get while using the package manager to install .NET Core.
+
+### Failed to fetch
+
+While installing the .NET Core package, you may see an error similar to `Failed to fetch ... File has unexpected size ... Mirror sync in progress?`. Generally speaking, this error means that the package feed for .NET Core is being upgraded with newer package versions, and that you should try again later. During an upgrade, the package feed should not be unavailable for more than 2 hours. If you continually receive this error for more than 2 hours, please file an issue at <https://github.com/dotnet/core/issues>.
```

Comment:
Hmm, maybe this should be in something we `INCLUDE`? I'm not familiar with the docs infra, just remembered it from the diff context. 🙂 This bit may change (without going away completely) as we figure out what we can do about this. (Also makes it easier to review.)

Reply:
Yes I think so. I started without the include because I thought it was only the ubuntu items 😁 I'll convert it.